pkg/runctl/controller.go

```diff
@@ -90,9 +96,31 @@ func NewController(factory k8s.ClientFactory, metrics metrics.Metrics) *Controll
 			controller.handleTektonTaskRun(new)
 		},
 	})
+
 	return controller
 }
 
+//meterCurrentPipelineStatus writes the current Duration of Pipeline States in a histogram
+func (c *Controller) meterCurrentPipelineStatus(ticker time.Duration) {
+	for range time.Tick(ticker) {
+		klog.V(4).Infof("Current metrics Observation still alive")
+		objs := c.pipelinerunStore.List()
+		for _, obj := range objs {
+			run, ok := obj.(*api.PipelineRun)
+			if !ok {
+				klog.V(4).Infof("Could not cast stored object to pipelinerun")
```

Comment:
The log message has no context information and thus is of little value. If that happens, it would be interesting what kind of object we got.